src/main/kotlin/dev/minn/jda/ktx/utils.kt

```diff
@@ -0,0 +1,24 @@
+package dev.minn.jda.ktx
+
+import net.dv8tion.jda.api.utils.MemberCachePolicy
+
+/**
+ * Allows you to do
+ * ```kotlin
+ * memberCachePolicy = MemberCachePolicy.ONLINE or MemberCachePolicy.VOICE or MemberCachePolicy.OWNER
+ * ```
+ * instead of
+ * ```kotlin
+ * memberCachePolicy = MemberCachePolicy.ONLINE.or(MemberCachePolicy.VOICE).or(MemberCachePolicy.OWNER)
+ * ```
+ *
+ * @param policy
+ * @return
```

Comment:
Docs?

Reply:
Just wanted to explain why this was added.

Reply:
wait do you _want_ docs or not want them?

Reply:
I want docs but not just `@param name` which is rather useless

Reply:
aight, will do.

I just left that there because intellij auto generate stuff.